<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1469</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Анализатор курса BTC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Априорная информация</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Ширина окна:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CSpinBox" name="m_spbxWinSize1">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Шаг:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CSpinBox" name="m_spbxStep1">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="CTimePlot" name="m_pricePlot" native="true"/>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Окно:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="m_spbxWinConst">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="7">
           <widget class="CQualityPlot" name="m_fixedWinPlot" native="true"/>
          </item>
          <item row="2" column="10" colspan="6">
           <widget class="CQualityPlot" name="m_fixedStepPlot" native="true"/>
          </item>
          <item row="0" column="10">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Шаг: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="11">
           <widget class="QSpinBox" name="m_spbxStepConst">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item row="0" column="19">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Окно от: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Шаг от: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="m_spbxStepStart">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>до: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="m_spbxStepEnd">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>инкремент: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QSpinBox" name="m_spbxStepIncrement">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="m_btnFixedWin">
            <property name="text">
             <string>Рассчитать</string>
            </property>
           </widget>
          </item>
          <item row="1" column="10">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Окно от: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="11">
           <widget class="QSpinBox" name="m_spbxWinStart">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
           </widget>
          </item>
          <item row="1" column="12">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>до: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="13">
           <widget class="QSpinBox" name="m_spbxWinEnd">
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="14">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>инкремент: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="15">
           <widget class="QSpinBox" name="m_spbxWinIncrement">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="12">
           <widget class="QPushButton" name="m_btnFixedStep">
            <property name="text">
             <string>Рассчитать</string>
            </property>
           </widget>
          </item>
          <item row="0" column="21">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>до: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="23">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>инкремент: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="20">
           <widget class="QSpinBox" name="m_winStart">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="22">
           <widget class="QSpinBox" name="m_winEnd">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="0" column="24">
           <widget class="QSpinBox" name="m_winStep">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="20">
           <widget class="QSpinBox" name="m_stepStart">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="22">
           <widget class="QSpinBox" name="m_stepEnd">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="24">
           <widget class="QSpinBox" name="m_stepStep">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="19">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Шаг от: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="21">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>до: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="23">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>инкремент: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="25">
           <widget class="QPushButton" name="m_btnCalcQuality">
            <property name="text">
             <string>Рассчитать</string>
            </property>
           </widget>
          </item>
          <item row="2" column="19" colspan="7">
           <widget class="Plot3D" name="m_3dPlot" native="true"/>
          </item>
          <item row="0" column="4" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="CSpinBox" name="m_spbxSm1">
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="CSpinBox" name="m_spbxSm2">
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_btnSmooth">
              <property name="text">
               <string>Сгладить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Анализ режима реального времени</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="2">
           <widget class="CSpinBox" name="m_spbxMa1Win">
            <property name="minimum">
             <number>60</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Окно: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="4">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Скользящее среднее 1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Шаг: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="CSpinBox" name="m_spbxMa1Step">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
          <item row="1" column="15">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="5" colspan="4">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Скользящая регрессия 1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Окно: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="7">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Шаг: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="CSpinBox" name="m_spbxMr1Win">
            <property name="minimum">
             <number>60</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
           </widget>
          </item>
          <item row="1" column="8">
           <widget class="CSpinBox" name="m_spbxMr1Step">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
           </widget>
          </item>
          <item row="0" column="9" colspan="6">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Скользящая регрессия N</string>
            </property>
           </widget>
          </item>
          <item row="1" column="9">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Окно 2: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="11">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Окно 3: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="10">
           <widget class="CSpinBox" name="m_spbxMr2Win">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="12">
           <widget class="CSpinBox" name="m_spbxMr3Win">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="13">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Окно 4: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="14">
           <widget class="CSpinBox" name="m_spbxMr4Win">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>86400</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="CTimePlot" name="m_indicatorPlot" native="true"/>
        </item>
        <item row="2" column="0">
         <widget class="CTimePlot" name="m_decisionPlot" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1469</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="m_actionLoadFile"/>
    <addaction name="separator"/>
    <addaction name="m_actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="m_actionLoadFile">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="m_actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTimePlot</class>
   <extends>QWidget</extends>
   <header>time_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CSpinBox</class>
   <extends>QSpinBox</extends>
   <header>utils.h</header>
  </customwidget>
  <customwidget>
   <class>CQualityPlot</class>
   <extends>QWidget</extends>
   <header>quality_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Plot3D</class>
   <extends>QWidget</extends>
   <header>quality_3dplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
